### DISCUSSION TITLE
Host

### DISCUSSION THREAD

**Discussion by ben-gibson** [ben-gibson/GitLink#172] [Polls]
Share which host you primarily use GitLink with.

**Comment by jpcmonster** [comment]
I chose github but should note it is github enterprise :)

**Comment by ravage84** [comment]
GitLab self-hosted (probably should have been a second option in the list)

**Comment by outdooracorn** [comment]
As I couldn't select multiple, and one was custom, here is a list of hosts I use most often (most frequently used first):
- Gerrit
- GitLab
- GitHub

**Comment by paour** [comment]
Github + Gitlab self-hosted

**Comment by dmitry-kovalev** [comment]
Gitlab self-hosted  + sometimes Github

**Comment by johntrandall** [comment]
gitlab, github

**Comment by vityakut** [comment]
Gitlab self-hosted + Github

**Comment by motofix** [comment]
Gerrit
You may add an option for it in the poll ;-)
